blues/application/tasks.py

```diff
@@ -224,24 +224,53 @@ def generate_nginx_conf(role='www'):
         f.write(conf)
 
 
-def notify_deploy(role=None):
+def get_github_org():
+    from .project import git_repository
+
+    url = git_repository().get('url', '')
+    if url or not url.startswith('git@github.com:'):
```

Comment:
I suppose it should be `if not url or not url.startswith`. You see what I mean when I say it's hard to read? :smiley_cat: